ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.qml

```diff
@@ -0,0 +1,484 @@
+    property var domainNetworkName: "ElectricDistribution"
+    property var tierName: "Medium Voltage Radial"
+    property var serviceCategoryName: "ServicePoint"
+    property var loadNetworkAttributeName: "Service Load"
+    property var phasesNetworkAttributeName: "Phases Current"
+    property var sampleStatus: 0
+
+    property bool reportHasRun: false
+
+    property var phases: ["A", "AB", "ABC", "AC", "B", "BC", "C", "DeEnergized", "Unknown"]
+    property var phaseQueue: []
+    property var currentPhase: null
+
+    property var checkedPhases: {
+        "A": false,
+        "AB": false,
+        "ABC": false,
+        "AC": false,
+        "BC": false,
+        "B": false,
+        "C": false,
+        "DeEnergized": false,
+        "Unknown": false
+    }
+
+    UtilityNetwork {
+        id: utilityNetwork
+        url: "https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer"
+
+        credential: Credential {
+            username: "editor01"
+            password: "S7#i2LWmYH75"
+        }
+
+        Component.onCompleted: {
+            utilityNetwork.load();
+        }
+
+        onLoadStatusChanged: {
+            if (utilityNetwork.loadStatus === Enums.LoadStatusLoaded) {
+                utilityAssetType = createUtilityAssetType();
+                utilityTier = createUtilityTier();
+
+                // Create a UtilityElement from the UtilityAssetType to use as the starting location
+                startingLocation = createStartingLocation();
+
+                // Get a default trace configuration from a tier in the network.
+                traceParams.traceConfiguration = createTraceConfiguration();
+
+                // Create a base condition to compare against
+                baseCondition = traceParams.traceConfiguration.traversability.barriers;
+
+                // Create a list of possible phases from a given network attribute
+                phaseCodedValuesList = createCodedValuesPhaseList();
+                sampleStatus = 2;
+            }
+        }
+
+        onTraceStatusChanged: {
+            if (traceStatus === Enums.TaskStatusCompleted) {
+                let customers = 0;
+                let load = 0;
+
+                traceResult.forEach(result => {
+                                        if (result.objectType === "UtilityElementTraceResult")
+                                            customers = result.elements.length;
+
+                                        else if (result.objectType === "UtilityFunctionTraceResult")
+                                            load = result.functionOutputs[0].result;
+                                    });
+
+                setGridText(currentPhase.name, customers, load);
+
+                if (phaseQueue.length > 0) {
+                    currentPhase = phaseQueue.pop();
+                    createReportForPhase(currentPhase);
+                } else {
+                    sampleStatus = 2;
```

Comment:
Ideally sampleStatus could make use of the relatively new QML feature of enumerations - https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#enumeration-attributes